### DISCUSSION TITLE
Why was the manual removed?

### DISCUSSION THREAD

**Discussion by hazu-89** [joaotavora/eglot#1091] [Q&A] [answered]
I decided to try eglot because of the recent news of it being merged into Emacs, but when I looked for the manual I found it was removed in https://github.com/joaotavora/eglot/commit/2d3e906b876f21bfaf7f34f705000085e6d97944. Why is that? Was it moved?

**Comment by joaotavora** [comment, marked as answer]
Good question.  I though the `README.md` answered it sufficiently, but maybe it doesn't.

The manual now also lives in Emacs.git.  You can find a published version here: https://joaotavora.github.io/eglot/

The manual's source is Emacs.git's `doc/misc/eglot.texi` file Here's [emacs-mirror's link to it](https://github.com/emacs-mirror/emacs/blob/master/doc/misc/eglot.texi).  You can proposed patches to `eglot.texi` just like you can to `lisp/progmodes/eglot.el`.

If you were somehow familiar with the previous `MANUAL.md` I would like to have your feedback about the manual rewrite/restructuring to Texinfo.  Is something missing or confusing in the new version?

**Reply by hazu-89** [reply]
> The manual now also lives in Emacs.git. You can find a published version here: https://joaotavora.github.io/eglot/

Thank you, this is what I was looking for! May I advice readding the `MANUAL.md` file with a link to this new manual? I found a few references to `MANUAL.md` in the search engine, but they led me to the 404 page.

I also completely missed the link to the new manual in the `README.md` file, so maybe it lacks visibility.

I'll read the manual and share my thoughts later tonight.

**Reply by joaotavora** [reply]
Ok. Your advice sounds good, so you can propose a PR with those changes to README.md and ressurecting MANUAL.md with a link to be the official one.
